### PR TITLE
Plan 37 expense card overview page

### PR DESCRIPTION
**Pull request by radmantvk**
This PR covers:
- Handling undefined% for cost in overview page
- Displaying data for expense card in the overview page

### CODE REVIEW

File: pro-planner-client/src/components/ExpenseSplit.jsx [hunk 1]
```diff
@@ -86,6 +84,22 @@ const ExpenseSplit = ({ className }) => {
     }
 
     processExpenses();
+    // useEffect(() => {
```
Comment (rikili) [inline pro-planner-client/src/components/ExpenseSplit.jsx:87]: please remove, large block of commented code.

File: pro-planner-client/src/components/ExpenseSplit.jsx [hunk 2]
```diff
@@ -117,10 +131,11 @@ const ExpenseSplit = ({ className }) => {
                                 <Col as="h5"> Availble </Col>
                                 <Col as="h5" className="text-end"> ${ budget - totalSpendings } </Col>
                             </Row>
+                            <>{console.log("spentToAvailableRatio " + spentToAvailableRatio)}</>
```
Comment (rikili) [inline pro-planner-client/src/components/ExpenseSplit.jsx:134]: please remove, console.log

File: pro-planner-client/src/routes/HomePage.jsx [hunk 3]
```diff
@@ -22,7 +25,12 @@ const HomePage = () => {
 
 	const isUserSelected = !!selectedUser;
 
+	const tripId = useLocation().pathname.split('/')[1];
```
Comment (rikili) [inline pro-planner-client/src/routes/HomePage.jsx:28]: I think you can use `planId` in place of `tripId`.

**Review by palijanii — APPROVED** [review]

File: pro-planner-client/src/routes/HomePage.jsx [hunk 4]
```diff
@@ -48,8 +52,9 @@ const HomePage = () => {
 				});
 			setParamStatus(LOAD_STATUS.LOADING);
 		}
-	}, [dispatch, planId, isUserSelected, navigate]);
-
+	}, [dispatch, planId, planId, isUserSelected, navigate]);
```
Comment (rikili) [inline pro-planner-client/src/routes/HomePage.jsx:55]: there's two `planId`s here

**Review by rikili — APPROVED** [review]
Looks good

**Pull request merged**